src/asciidoc/docs/develop/editor.adoc

```diff
@@ -1,123 +1,156 @@
-The following types are available.
+When you fork an existing project or scaffold:
+[.singlespaced]
+* The Git repository of that project/scaffold is copied to your account.
+* A new project is created and opened in the Code Editor.
 
-=== Default
+When you use the Code Editor to publish your Koji, the Code Editor commits your changes and pushes the updated files to the `master` branch.
+This push triggers the deployment process.
 
-Selecting *Default* shows a running preview of your template.
 
-=== Remix
+.Using other editors
+****
+You can use other editors instead of the Koji Code Editor and still access your Git repository.
 
-Selecting *Remix* shows what remixers see when customizing values.
+To determine the URL of your project's Git repository:
+[.singlespaced]
+. Open your project in the Code Editor.
+. Navigate to *Advanced > Repository Info*.
 
-=== More
+Learn more: <<use-git#_cloning_the_git_repository>>
+****
+
+{blank}
+
+[IMPORTANT]
+====
+If you made changes to your project files xref:work-locally.adoc[outside the Koji editor], you must update the project files before editing them in the Code Editor again.
+[.singlespaced]
+. In the Koji Code Editor, expand the terminal pane.
+. Run `git pull`.
+====
 
-Selecting *More* displays a menu of additional previews.
 
+== Previewing your Koji
+
+You can select what is displayed in the preview pane.
+
+* *Default* - Displays a fully functional preview of your Koji.
+
+* *Remix* - Displays what remixers see when customizing your Koji.
+
+* *More* - Displays a menu of additional previews.
++
 image::more.png[More menu]
++
+** *About* - Displays an About page, if you enabled the associated entitlement.
+
+** *Admin* - Displays an Admin page, if you enabled the associated entitlement.
+
+** *Receipt (seller)* - Displays a preview of the transaction receipt for the seller, if you implemented in-app purchases and enabled the dynamic receipts entitlement.
 
-* *About* - Displays an About page, if you've enabled the associated entitlement.
+** *Receipt (buyer)* - Displays a preview of the transaction receipt for the buyer, if you implemented in-app purchases and enabled the dynamic receipts entitlement.
 
-* *Admin* - Displays an Admin page, if you've enabled the associated entitlement.
+** *Sticker* - Displays how your Koji will look if it's imbedded in another Koji.
 
-* *Receipt (seller)* - Displays a preview of the transaction receipt for the seller, if you've implemented in-app purchases and enabled the dynamic receipts entitlement.
+** *Thumbnail* - Displays how your Koji will look if it's shared on social media.
 
-* *Receipt (buyer)* - Displays a preview of the transaction receipt for the buyer, if you've implemented in-app purchases and enabled the dynamic receipts entitlement.
+To learn more about using previews, see <<testing-templates#_testing_in_the_koji_editor,Testing in the Koji editor>>.
```

Comment:
I would move this to the top of this section and then add another subsection for Previews in the code editor. That way, we can keep the concise summary info here for the various preview and debugging modes, but point to the comprehensive reference that covers all of these topics in more detail.

Reply:
I reviewed the testing page that it is pointing to, and the information is actually duplicated.
- In the code editor page, removed the link.
- In the testing page, removed the duplicate information and added a link to the code editor page instead.